### PR TITLE
Hotfix/test update

### PR DESCRIPTION
**Pull request by bfloreaDVSA**
let's see...

### CODE REVIEW

File: src/main/java/clients/TestResultsClient.java [hunk 1]
```diff
@@ -265,12 +266,12 @@ public Response callPostTestResults(Object object) {
...
 
         Response response = given()
-//                .log().all()
                 .filters(new BasePathFilter())
                 .contentType(ContentType.JSON)
                 .pathParam("vin", vin)
+                .log().all()
                 .get("/test-results/{vin}");
-
+        System.out.println("results: " + response.then().log().all());
```
Comment (jgimber-deloitte) [inline src/main/java/clients/TestResultsClient.java]: Is this a duplicate of the log().all() on line 272?

Comment (bfloreaDVSA) [reply src/main/java/clients/TestResultsClient.java]: fixed

File: src/test/java/testresults/PostTestResultsNegMainLvlCancelled.java [hunk 2]
```diff
@@ -701,7 +716,8 @@ public void testResultsIntegerVehicleType() {
         testResultsSteps.validatePostErrorData("vehicleType", "must be one of [psv, hgv, trl]");
     }
 
-
+    //TODO - possible problem with gateway
```
Comment (jgimber-deloitte) [inline src/test/java/testresults/PostTestResultsNegMainLvlCancelled.java]: These Ignores... is there a related issue for this?
e.g. should you be using the @Issue tag for this, to link the ignores to the problem?

File: src/test/java/activities/GetActivities.java [hunk 3]
```diff
@@ -94,42 +97,42 @@ public void postActivitiesActivityTypeVisit212() {
...
     @Title("CVSB- / CVSB- - AC7 ")
     @Test
     public void postActivitiesActivityTypeWait() {
-        activitiesSteps.postActivities(activitiesData.setActivityType("wait").build());
+        activitiesSteps.postActivities(ActivitiesData.buildActivitiesIdData().setActivityType("visit").build());
+        String parentId =  activitiesSteps.checkAndGetResponseId();
+        activitiesSteps.postActivitiesWithWaitReason(ActivitiesData.buildActivitiesParentIdData().setParentId(parentId).setActivityType("wait").build());
         activitiesSteps.statusCodeShouldBe(201);
-        activitiesSteps.getActivities("wait", null, null, DataUtil.buildCurrentDateTime(-1), null);
+        activitiesSteps.getActivities("wait", null, null, "2018-02-13", "2018-02-13");
```
Comment (dpanzaru-deloitte) [inline src/test/java/activities/GetActivities.java]: These dates should not be hardcoded.

Comment (bfloreaDVSA) [reply src/test/java/activities/GetActivities.java]: thanks, fixed !

**Pull request merged**